### PR TITLE
Fixed stray comma.

### PR DESCRIPTION
**Pull request by MC42**

### CODE REVIEW

**Comment by fangeugene** [comment]
I think there's a bigger issue than just a stray comma. Can you fix and test the different variations? For example:
1114 says: `Home Championship: St. Louis, 2017 | , Detroit, 2018+`
254 says: `Home Championship: Houston, 2017 | +`

**Comment by MC42** [comment]
I'll look into it shortly.  Sorry :sweat_smile:

**Comment by MC42** [comment]
@fangeugene I think I fixed it.  Turns out I had kept the + and the pipe *outside* of the logic check, so now (in theory...) it should work properly.

File: templates_jinja2/team_partials/team_info.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
       {% if team.name %}<span class="glyphicon glyphicon-info-sign"></span> aka <em>{{ team.name }}</em><br>{% endif %}
       {% if team.rookie_year %}<span class="glyphicon glyphicon-calendar"></span> First competed in {{ team.rookie_year }}<br>{% endif %}
       {% if team.website %}<span class="glyphicon glyphicon-globe"></span> <a href="{{team.website}}" target="_blank">{{ team.website }}</a><br>{% endif %}
-      {% if team.championship_location %}<span class="glyphicon glyphicon-flag"></span> Home Championship: {{team.championship_location.2017}}, 2017 | {% if team.championship_location.2017 != team.championship_location.2018 %}, {{team.championship_location.2018}}, 2018{% endif %}+<br>{% endif %}
+      {% if team.championship_location %}<span class="glyphicon glyphicon-flag"></span> Home Championship: {{team.championship_location.2017}}, 2017 {% if team.championship_location.2017 != team.championship_location.2018 %} {{team.championship_location.2018}} | , 2018+ {% endif %}<br>{% endif %}
```
Comment (fangeugene) [inline templates_jinja2/team_partials/team_info.html]: I believe the pipe should go before `{{team.championship_location.2018}}`?

**Pull request merged**